.github/maintainers_guide.md

```diff
@@ -1,199 +1,162 @@
-# Maintainers Guide
+# Maintainer's Guide
 
-> ⚠️ This maintainers guide is out of date. Progress on updating it is being [tracked](https://github.com/slackapi/node-slack-sdk/issues/748).
+This document describes tools, tasks and workflow that one needs to be familiar with in order to effectively maintain this project. If you use this package within your own software but don't plan on modifying it, this guide is **not** for you. Please refer to the [Contributor's Guide](https://github.com/slackapi/node-slack-sdk/blob/main/.github/contributing.md).
 
-This document describes tools, tasks and workflow that one needs to be familiar with in order to effectively maintain
-this project. If you use this package within your own software but don't plan on modifying it, this guide is
-**not** for you.
+## 🛠 Tools
+Maintaining this project requires installing [Node.js](https://nodejs.org). All of the remaining tools are downloaded as `devDependencies`, which means you'll have them available once you run `npm install` in a working copy of this repository.
```

Comment:
Great question! The vulnerabilities look to be all associated with `lerna`. Since we're moving towards removing `lerna` all together, I don't believe it's worth fixing it beforehand, but if any if any of the other maintainers disagree on that point, we should definitely create an issue to address it.